packages/webapp/src/inductions/components/induction-journeys/inviter-witness-journey.tsx

```diff
@@ -0,0 +1,193 @@
+import React, { Dispatch, SetStateAction, useState } from "react";
+import { Heading, Link, Text, useIsCommunityActive, useUALAccount } from "_app";
+import { MemberData } from "members";
+import { convertPendingProfileToMemberData } from "inductions";
+import {
+    InductionVideoFormContainer,
+    InductionEndorsementForm,
+    InductionVideoSubmitConfirmation,
+} from "./inviter-witnesses";
+import {
+    EndorsementsStatus,
+    InductionExpiresIn,
+    InductionJourneyContainer,
+    MemberCardPreview,
+    WaitingForProfile,
+} from "inductions/components";
+import { Endorsement, Induction, InductionStatus } from "inductions/interfaces";
+import { InductionStepGenesis, InductionStepInviter } from "./common";
+
+interface ContainerProps {
+    step: InductionStepInviter | InductionStepGenesis;
+    memberPreview?: MemberData;
+    children: React.ReactNode;
+}
+
+const Container = ({ step, memberPreview, children }: ContainerProps) => (
+    <>
+        <InductionJourneyContainer step={step}>
+            {children}
+        </InductionJourneyContainer>
+        {memberPreview && <MemberCardPreview memberData={memberPreview} />}
+    </>
+);
+
+const RecommendReview = ({
```

Comment:
can we push it to the end of the file or extract it to its own component? it seems unnatural to open the file and do not see the main File component in it (`InviterWitnessJourney`).

Reply:
Yeah, I can push it to the end of the file. If more than just this file is referencing it, I'll break it out into its own file.